input.py

```diff
@@ -0,0 +1,32 @@
+from typing import Counter
+from unittest.result import failfast
+from spellchecker import SpellChecker
+
+
+def function(sting):

```

Comment:
This should have a name that reflect what it does:
```suggestion
def get_symptoms_from_input(sting):
```